config_examples/dag_v4v6_trm_anycastrp/host_vars/node_vars/Leaf-02.yml

```diff
@@ -66,3 +66,23 @@ overlay_interfaces:
     subnet_mask: '255.255.255.0'
     loopback: 'yes'
     vrf: 'blue'
+  
+  Loopback901:
+    name: 'Vrf Loopback'
+    ip_address: '10.1.255.2'
+    subnet_mask: '255.255.255.255'
+    ipv6:
+      - 'FC00:1:255::2/128'
+    loopback: 'yes'
+    pim_enable: 'yes'
+    vrf: 'green'
+
+  Loopback902:
+    name: 'Vrf Loopback'
+    ip_address: '10.2.255.2'
+    subnet_mask: '255.255.255.255'
+    ipv6:
+      - 'FC00:2:254::2/128'
```

Comment:
i propose to have ipv6 FC00:2:255::2/128 to have consistency with ipv4 address 10.2.255.2/32